crates/amaru/src/bin/amaru/cmd/dump_chain_db.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 PRAGMA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use amaru_kernel::network::NetworkName;
+use amaru_kernel::string_utils::ListToString;
+use amaru_kernel::{Header, to_cbor};
+use amaru_ouroboros_traits::{ChainStore, IsHeader};
+use amaru_stores::rocksdb::consensus::RocksDBStore;
+use clap::{Parser, arg};
+use std::fmt::Display;
+use std::sync::Arc;
+use std::{error::Error, path::PathBuf};
+
+#[derive(Debug, Parser)]
+pub struct Args {
```

Comment:
For later: add flags to select what you want to dump (headers, blocks, relations...)